llama/cli.py

```diff
@@ -12,6 +13,8 @@
 @click.group()
 @click.pass_context
 def cli(ctx):
+    if config.SENTRY_DSN:
```

Comment:
I don't think we need to check for the `SENTRY_DSN` here, if you call `sentry_sdk.init()` with the dsn param as `None` (the default config value) it will just do nothing.